src/instructlab/model/backends/vllm.py

```diff
@@ -202,6 +203,25 @@ def run_vllm(
     return vllm_process, tmp_files
 
 
+def is_bnb_quantized(model_path: str) -> bool:
+    """
+    Check if provided model has quantization config with bitsandbytes specified.
+
+    Args:
+        model_path    (Path):         The path to the model files
+
+    Returns:
+        bool: Whether or not the model has been quantized via bitsandbytes
+    """
+    with open(os.path.join(os.fspath(model_path), "config.json"), "r") as config_json:
+        config = json.load(config_json)
+
+    return (
+        "quantization_config" in config
+        and config["quantization_config"]["quant_method"] == "bitsandbytes"
```

Comment:
Change this line to use `.get` in case it's not there for some reason:
```suggestion
        and config["quantization_config"].get("quant_method", "") == "bitsandbytes"
```